eth/tracers/internal/tracetest/testdata/call_tracer_parity/deep_calls.json

```diff
@@ -120,7 +120,8 @@
       },
       "blockNumber": 25001,
       "result": {
-        "gasUsed": "0x12bb3"
+        "gasUsed": "0x12bb3",
+        "output": "0x"
```

Comment:
`0x`'s are in! Do docs need updating?

Reply:
No we don't. They were in already under curtain conditions, while now it's the default, except some conditions.